openapi/components/schemas/Rules/Actions/adjust-ready-to-pay.yaml

```
@@ -0,0 +1,107 @@
+allOf:
+  - $ref: ../RuleAction.yaml
+  - type: object
+    description: Add risk score
+    properties:
+      gatewayAccountsMode:
+        type: string
+        description: |
+          The global gateway accounts mode that will be applied at the beginning of the action processing.
```

```suggestion
          The global gateway account mode that will be applied at the beginning of the action processing.
```
Not entirely sure about this